### PR TITLE
MINOR: Link to related class in the Admin client documentation

### PR DESCRIPTION
**Pull request by jsancio**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by jsancio** [comment]
Verified that the correct Java Doc was generated.

File: core/src/main/scala/kafka/controller/ControllerContext.scala [hunk 1]
```diff
@@ -37,6 +37,12 @@ object ReplicaAssignment {
   }
 }
 
+
+/**
+ * @param replicas the seq of brokers assigned to the partition
+ * @param addingReplicas the replicas that were added if there is a pending reassignment
```
Comment (hachikuji) [inline core/src/main/scala/kafka/controller/ControllerContext.scala]: nit: "were added" -> "are being added"?

File: core/src/main/scala/kafka/controller/ControllerContext.scala [hunk 2]
```diff
@@ -37,6 +37,12 @@ object ReplicaAssignment {
   }
 }
 
+
+/**
+ * @param replicas the seq of brokers assigned to the partition
```
Comment (hachikuji) [inline core/src/main/scala/kafka/controller/ControllerContext.scala]: Maybe worth mentioning that this list includes both the adding and removing replicas?

**Comment by cmccabe** [comment]
Looks good overall

File: core/src/main/scala/kafka/controller/ControllerContext.scala (hunk 1 above)
Comment (jsancio) [reply core/src/main/scala/kafka/controller/ControllerContext.scala]: Done.

File: core/src/main/scala/kafka/controller/ControllerContext.scala (hunk 2 above)
Comment (jsancio) [reply core/src/main/scala/kafka/controller/ControllerContext.scala]: Done.

**Comment by jsancio** [comment]
Thanks for the review @cmccabe and @hachikuji. Added a commit based on your feedback.

**Review by hachikuji — APPROVED** [review]
LGTM

**Comment by hachikuji** [comment]
I will go ahead and merge this since we are just modifying the comments. I verified locally docs built locally.

**Pull request merged**